ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -399,24 +399,54 @@ public void testAndAssert(RouterErrorCode expectedError) throws Exception {
   }
 
   /**
-   * Test the case with Blob_Not_Found errors from most servers, and Blob_Deleted at just one server. The latter
-   * should be the exception received for the operation.
+   * Test the case with Blob_Not_Found errors from most servers, and Blob_Deleted, Blob_Expired or
+   * Blob_Authorization_Failure at just one server. The latter should be the exception received for the operation.
    * @throws Exception
    */
   @Test
-  public void testErrorPrecedenceWithBlobDeletedAndExpiredCase() throws Exception {
+  public void testErrorPrecedenceWithSpecialCase() throws Exception {
     doPut();
     Map<ServerErrorCode, RouterErrorCode> serverErrorToRouterError = new HashMap<>();
     serverErrorToRouterError.put(ServerErrorCode.Blob_Deleted, RouterErrorCode.BlobDeleted);
     serverErrorToRouterError.put(ServerErrorCode.Blob_Expired, RouterErrorCode.BlobExpired);
+    serverErrorToRouterError.put(ServerErrorCode.Blob_Authorization_Failure, RouterErrorCode.BlobAuthorizationFailure);
     for (Map.Entry<ServerErrorCode, RouterErrorCode> entry : serverErrorToRouterError.entrySet()) {
       Map<ServerErrorCode, Integer> errorCounts = new HashMap<>();
       errorCounts.put(ServerErrorCode.Blob_Not_Found, replicasCount - 1);
       errorCounts.put(entry.getKey(), 1);
+      System.out.println(errorCounts);
```

Comment:
Do we need to print it out? If not, we can remove this line.

Reply:
No. This was deleted. For some reason, rebase is not taking this change in. 
A new commit was added and also addressed Enum comment. Good point! Thanks!